### ISSUE TITLE
jekyll?

### ISSUE THREAD

**Issue by infominer33** [typesense/gatsby-plugin-typesense#5]
I'm wondering about adapting this for jekyll, and a little confused about it being a "Gatsby" plugin.

If it's just looking for `data-typesense-field` in html documents, is there any reason I can't just add these to my [*any website builder*] template, and generate the index in this way?

**Comment by jasonbosco** [comment]
Hi @infominer33! Yes, you can certainly do that. In fact, I was recently thinking about extracting the portion of this package that looks for data attributes into a separate package, so it can be used in exactly the way you describe against any site. Would that be useful?

The gatsby-esque portions of this particular plugin come from the fact that it hooks into the gatsby build system and runs automatically post build.

**Comment by infominer33** [comment]
yes! that would be so great! I don't mind managing the integration from my side of things... I don't know much about javascript so would take quite a bit for me to figure how to pull apart that code.
